springfox-spring-web/build.gradle

```diff
@@ -11,6 +11,7 @@ ext {
 
 dependencies {
   compile "org.reflections:reflections:$reflections"
+  compile files("${System.properties['java.home']}/../lib/tools.jar")
```

Comment:
If you want this in a different repo that's fine. I can create a new repo in my github account. That would be better for me since I will use Maven to build the modules.